src/sweeps/config/schema.json

```
@@ -286,6 +286,30 @@
       },
       "additionalProperties": false
     },
+    "param_inv_loguniform": {
+      "type": "object",
+      "description": "inverse log uniform distribution",
+      "required": [
+        "distribution",
+        "max",
+        "min"
+      ],
+      "properties": {
+        "min": {
+          "description": "float",
+          "type": "number"
+        },
+        "max": {
+          "description": "float",
+          "type": "number"
```

```suggestion
          "type": "number",
          "exclusiveMinimum": 0
```
to avoid any variables that are less than or equal to zero, which will lead to floating point errors in the log(1 / x) calculation